common/tools/eslint-plugin-azure-sdk/src/rules/ts-package-json-homepage.ts

```diff
@@ -37,7 +37,7 @@ export = {
             const nodeValue = node.value as Literal;
 
             if (
-              !/^https:\/\/github.com\/Azure\/azure-sdk-for-js\/(blob|tree)\/master\/sdk\/(([a-z]+-)*[a-z]+\/)+(README\.md)?$/.test(
+              !/^https:\/\/github.com\/Azure\/azure-sdk-for-js\/(blob|tree)\/master\/sdk(\/([a-z]+-)*[a-z]+)+(\/README\.md)?$/.test(
```

Comment:
Existing regex was enforcing either a trailing slash OR a `README.md` so that these were valid:

```
https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/appconfiguration/app-configuration/README.md
https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/appconfiguration/app-configuration/
```

But this was not:

```
https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/appconfiguration/app-configuration
```

Reply:
@ramya-rao-a do you have a preference?

Reply:
FYI I ended up restoring the previous linter behavior so resolving this conversation